README.md

```diff
@@ -19,7 +19,7 @@ In order to obtain the openshift installer and client, visit [https://origin-rel
 Use `oc` to download and extract the tools from the image:
 
 ```
-$ oc adm release extract --tools registry.svc.ci.openshift.org/origin/release:4.4.0-0.okd-2020-01-20-084618
+$ oc adm release extract --tools registry.svc.ci.openshift.org/origin/release:4.4
```

Comment:
```suggestion
$ oc adm release extract --tools registry.svc.ci.openshift.org/origin/release:4.4.0-0.okd-2020-01-28-022517
```

We should recommend using Preview 2 release since its more stable than very recent 4.4 code

Reply:
@vrutkovs, I'm noticing that the images are getting constantly updated. Are the starred ones (*) the only "stable" ones?

Also, do you need anything from my side on this PR? I might have missed a step in your process.

Reply:
>I'm noticing that the images are getting constantly updated

Correct, these are tested on CI, but due to insufficient coverage these might have a lot of bugs. We shouldn't be recommending those in the README

>Are the starred ones (*) the only "stable" ones?

Starred ones won't be GC'ed ("nightly" images would be kept for 48hrs). So far we have only one pinned image - Preview 2, which we should recommend in README until further notice

>Also, do you need anything from my side on this PR

Commit the suggestion to use Preview 2 payload

Reply:
Commited your suggestion. Sorry for the confusion!